radio/src/telemetry/mavlink/mavlink_telem_mavapi.cpp

```diff
@@ -0,0 +1,139 @@
+/*
+* (c) www.olliw.eu, OlliW, OlliW42
```

Comment:
Please change the boiler plates to match the normal GPLv2 boiler plate **everywhere in your code**.